PROVISIONING.md

```diff
@@ -161,25 +161,53 @@ Use case: large scale deployments (where automation and security are major drive
   - Comment: probably need to support multiple preferences by operators. some facilities have to be actually local, other can be proxied to a centralized location, others yet can be completely centralized without a local presence.
-- Device contacts the bootstrap server to get the certificate, bootstrap server will facilitate the request towards CA
-  - Take a look at SCEP (Simple Certificate Enrollment Protocol)
-  - The communication with bootstrap server doesn't have to be secure at this point
-  - Question: preconfigured certificates? how they distributed? what is alternative to certificates? PKI based crypto ?
-- If we want to protect stolen/mistaken shiipment (device needs to authenticate network) we have to use Vouchers
-  - Take a look at RFC 8366 - A Voucher Artifact for Bootstrapping Protocols
+- Need to consider mDNS as another option if DHCP is absent.
+- Need to consider LLDP/SSDP as well.
+
+##### Network trusts DPU
+
+- Device establishes an HTTPS connection with the "Bootstrap Server"
+  - Question: using what certificates?
   - More info is here <https://github.com/opiproject/opi-prov-life/blob/main/architecture/Zero-Touch-Provisioning%E2%80%94Approaches-to-Network-Layer-Onboarding.pdf>
-- Device can now establish an HTTPS connection with the bootstrap server using certificates from above
-  - Question: one way or mutual (two-way) authentication is required ?
-  - Question: three-way trust established here, between device identity, manufacturer, and operator ???
+- Device sends a request to a "Bootstrap Server" to join the network by providing its IDevID.
+- "Bootstrap Server" decides to accept debvice to the network or reject
+  - Needs trust anchor of IDevID
+  - Takes "Voucher Server" URL from IDevID
+    - <https://www.rfc-editor.org/rfc/pdfrfc/rfc8366.txt.pdf> (A Voucher Artifact for Bootstrapping)
+  - Fetches Voucher with trust anchor of IDevID inside the voucher
+  - Verifies IDevID and accepts Device to the network
+
+##### DPU trusts network
+
+- Device has to decide to trust/join the network now
+  - Device asks "Bootstrap Server" to give a Voucher
+  - Device starts Voucher verification
+    - Device verifies the signature of the voucher
+    - Device verifies IDevID from the Voucher belongs to the device
+    - Device verifies the correctness of the network by comparing Voucher network root certificate to TLS certificate from the initial connection to the "Bootstrap Server"
+  - Device trusts the network now
+- Device needs LDevID (crypto identity in the local domain)
+  - Device asks "Bootstrap Server" to get "CA Server" address
+  - Device generates keypair and CSR
+  - Device sends CSR to "CA Server" (SCEP -Simple Certificate Enrollment Protocol)
+  - "CA Server" returns certificate to the Device
+  - Device has now cryptograhic identity in the lcoal domain
+- Question: three-way trust established here, between device identity, manufacturer, and operator ???
```

Comment:
i think we can remove this question, it's moot for SZTP which covers this area well.